pkg/admissioncontroller/webhooks/auth/seed/graph/eventhandler_managedseed.go

```diff
@@ -44,31 +42,11 @@ func (g *graph) setupManagedSeedWatch(ctx context.Context, informer cache.Inform
 		},
 
 		UpdateFunc: func(oldObj, newObj interface{}) {
-			oldManagedSeed, ok := oldObj.(*seedmanagementv1alpha1.ManagedSeed)
-			if !ok {
-				return
-			}
-
 			newManagedSeed, ok := newObj.(*seedmanagementv1alpha1.ManagedSeed)
 			if !ok {
 				return
 			}
-
-			oldSeedTemplate, _, err := seedmanagementv1alpha1helper.ExtractSeedTemplateAndGardenletConfig(oldManagedSeed)
-			if err != nil {
-				return
-			}
-			newSeedTemplate, _, err := seedmanagementv1alpha1helper.ExtractSeedTemplateAndGardenletConfig(newManagedSeed)
-			if err != nil {
-				return
-			}
-
-			if oldManagedSeed.Spec.Shoot.Name != newManagedSeed.Spec.Shoot.Name ||
-				!gardencorev1beta1helper.SeedBackupSecretRefEqual(oldSeedTemplate.Spec.Backup, newSeedTemplate.Spec.Backup) ||
-				!apiequality.Semantic.DeepEqual(oldSeedTemplate.Spec.SecretRef, newSeedTemplate.Spec.SecretRef) ||
-				!gardenletBootstrapModeEqual(oldManagedSeed.Spec.Gardenlet, newManagedSeed.Spec.Gardenlet) {
-				g.handleManagedSeedCreateOrUpdate(ctx, newManagedSeed)
-			}
```

Comment:
Why was this code removed? It doesn't seem to be related to the rest of the change, did you realize it's not needed, or perhaps that the checks were not really necessary?

Reply:
When the `gardener.cloud/operation=reconcile` annotation is added to the `ManagedSeed` (e.g., by a human (manually) or by the seed lifecycle controller in GCM) then these checks would have prevented the `handleManagedSeedCreateOrUpdate` from being called.